homeassistant/components/motion_blinds/__init__.py

```diff
@@ -43,20 +43,18 @@ class DataUpdateCoordinatorMotionBlinds(DataUpdateCoordinator):
 
     def __init__(
         self,
-        hass,
-        logger,
-        coordinator_info,
+        hass: HomeAssistant,
+        logger: logging.Logger,
+        coordinator_info: dict[str, Any],
         *,
-        name,
-        update_interval=None,
-        update_method=None,
```

Comment:
I was going to add hints to `update_method`, but since it never gets set I figured it was easier to simply remove it.